### DISCUSSION TITLE
How to send match data to specific user?

### DISCUSSION THREAD

**Discussion by nteko** [heroiclabs/nakama-dart#49] [Q&A] [answered]
I tried to send match data to a specific user or group.
According to the document, it's possible, but is it not implemented in this SDK yet?
and is there way to broadcast match data include me(sender)?
One more thing, is there way to refesh session token?
Thank you.

**Comment by obrunsmann** [comment, marked as answer]
The respective methods to send and receive data for a match are `sendMatchData` and `onMatchData`. 

> One more thing, is there way to refesh session token?

Yep:
```dart
Future<model.Session> sessionRefresh({
    required model.Session session,
    Map<String, String>? vars,
  });
```
From the nakama client
